transform/all_collection/spark/transformations/commons.py

```diff
@@ -146,18 +151,34 @@ def harvest_best_access_right(
     return df.drop(BEST_ACCESS_RIGHT)
 
 
-def simplify_language(df: DataFrame) -> DataFrame:
-    """Simplify language - get only label and convert structure to a string"""
-    return df.withColumn("language", col("language")["label"])
-
-
 def create_open_access(best_access_right: List, harvested_properties: Dict) -> None:
     """Create boolean value whether record is open access or not, based on col_name"""
     open_access_column = [bool(access == _OPEN_ACCESS) for access in best_access_right]
 
     harvested_properties[OPEN_ACCESS] = open_access_column
 
 
+def map_publisher(df: DataFrame) -> DataFrame:
+    """Map publishers' value for OAG resources"""
+    # Values are mapped to the keys
+    mapping = {
+        ZENODO: "ZENODO",
+        FIGSHARE: "figshare",
+    }
```

Comment:
You may consider extracting the `mapping` variable outside. It could help you in the future to update the logic faster.